### DISCUSSION TITLE
Too many login attempts.

### DISCUSSION THREAD

**Discussion by Jtinator** [runelite/runelite#16377] [Q&A] [answered]
Hi, I can not log in via runelite due to too many login attempts it says. I can log in on mobile how can I fix this?

**Comment by Felanbird** [comment, marked as answer]
You can work around this issue by installing the Jagex Launcher from https://www.jagex.com/en-GB/launcher and then using it to launch RuneLite.

In the future join the discord for support instead of using Github
![image](https://user-images.githubusercontent.com/41973452/225169736-89b6877c-b546-43e4-9247-cc390f983ef1.png)
